CHANGELOG.md

```diff
@@ -52,6 +52,13 @@ We have several packages which live in this repository. Changes are tracked sepa
 
 ### [defmt-next]
 
+* [#960]: Fix `Format` not accepting multiple helper attribute instances
```

Comment:
Oh, I just noticed this seems to be a bad merge resolution. I believe we just need the #983 line at the bottom.